security/backends/elasticsearch/tests.py

```diff
@@ -21,7 +22,11 @@ def enable(self):
         for document_class in (CommandLog, CeleryTaskRunLog, CeleryTaskInvocationLog,
                                InputRequestLog, OutputRequestLog):
             document_class._index._name = f'{uuid}.{document_class._index._name}'
-            document_class.init()
+            if issubclass(document_class, PartitionedLog):
+                template = document_class.get_template()
+                template.save()
```

Comment:
asi to není MR na mě, ale pozor na toto https://github.com/elastic/elasticsearch-dsl-py/issues/1576, ať to zbytečně neděláte 2x. legacy templates jsou depricated. Myslím, že toto ti vytvoří legacy

Reply:
Ahoj, vycházel jsem z dokumentace https://elasticsearch-dsl.readthedocs.io/en/latest/persistence.html#indextemplate, ale mrknu na to :-)